### PR TITLE
Add  sample MySQL script for database integration guides

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Add sample MySQL Script for querying_mysql_database guide.

### CODE REVIEW

File: docs/content/src/learn/tutorials/database-integrations/querying-mysql-database/src/querying_mysql_database/resources/sampleScript.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+INSERT INTO Employee VALUES ('0001', 'thomas', 'collins',32,'thomasc@enp.com'),
+                            ('0002', 'henry', 'parker',22,'henryp@enp.com'),
+                            ('0003', 'abner', 'collins',40,'abner@enp.com'),
+                            ('0004', 'anne', 'clements',38,'annec@enp.com'),
+                            ('0005', 'thomas','kirk',25,'thomask@enp.com'),
+                            ('0006','emeline','foultaon',31,'emelinef@enp.com'),
+                            ('0007', 'jared', 'morris',32,'jaredm@enp.com'),
+                            ('0008', 'henry', 'fosterr',37,'henryf@enp.com'),
+			        ('0009', 'jones', 'kirk',32,'jonesk@enp.com'),
```
Comment (pramodya1994) [inline docs/content/src/learn/tutorials/database-integrations/querying-mysql-database/src/querying_mysql_database/resources/sampleScript.sql]: Alignment issue

File: docs/content/src/learn/tutorials/database-integrations/querying-mysql-database/src/querying_mysql_database/resources/sampleScript.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#Sample script for Ballerina Integrator Inserting into MySQL Database
```
Comment (pramodya1994) [inline docs/content/src/learn/tutorials/database-integrations/querying-mysql-database/src/querying_mysql_database/resources/sampleScript.sql]:
```suggestion
# Sample script for Ballerina Integrator Inserting into MySQL Database 
```

Comment (pramodya1994) [reply docs/content/src/learn/tutorials/database-integrations/querying-mysql-database/src/querying_mysql_database/resources/sampleScript.sql]: And please check whether this sentence is grammatically correct

Comment (Nipunich) [reply docs/content/src/learn/tutorials/database-integrations/querying-mysql-database/src/querying_mysql_database/resources/sampleScript.sql]: Fixed

**Pull request merged**